[DOC]Restructuring (#3890)
pachyderm/pachyderm: doc/concepts/pipeline-concepts/job.md

# Job

> A Pachyderm job is a unit of work that a pipeline triggers

Reviewer: ```suggestion
A Pachyderm job is an execution of a pipeline that triggers
```

> when it detects new data in the input repository. Each

Reviewer: ```suggestion
when new data is detected in an input repository. Each
```

job runs your code against the current commit and

> then submits the results to the output repository. A pipeline

Reviewer: ```suggestion
then submits the results to the output repository and creates a single output commit. A pipeline
```

> triggers a new job every time you submit new changes into your

Reviewer: ```suggestion
triggers a new job every time you submit new changes, a commit, into your
```

input source.

Each job has the following stages:

| Stage     | Description  |
| --------- | ------------ |

> | Starting  | Pachyderm starts the job when it detects new data in the input repository. <br> The new data appears as a commit in the input repository, and Pachyderm <br> automatically launches the pipeline. Pachyderm spins the number of Pachyderm worker pods <br> specified in the pipeline spec and spreads the workload among them. |

Reviewer: ```suggestion
| Starting  | Pachyderm starts the job when it detects new data in the input repository. <br> The new data appears as a commit in the input repository, and Pachyderm <br> automatically launches the job. Pachyderm spins the number of Pachyderm worker pods <br> specified in the pipeline spec and spreads the workload among them. |
```

> | Running   | `pachd` runs the transformation code that is specified <br> in the pipeline specification against the newly committed data. |

Reviewer: ```suggestion
| Running   | Pachyderm runs the transformation code that is specified <br> in the pipeline specification against the data in the input commit. |
```

> | Merging   | Pachyderm concatenates the results of the processed <br> data into file or files and uploads them to the output repository. |

Reviewer: ```suggestion
| Merging   | Pachyderm concatenates the results of the processed <br> data into one or more files, uploads them to the output repository, completes the final output commits, and creates/persists all the versioning metadata |
```

Reviewer: You'll need to fix the "< br >" probably with the content changes.
